services/datamanager/builtin/builtin.go

```diff
@@ -115,11 +117,22 @@ var viamCaptureDotDir = filepath.Join(os.Getenv("HOME"), "capture", ".viam")
 
 // NewBuiltIn returns a new data manager service for the given robot.
 func NewBuiltIn(_ context.Context, r robot.Robot, _ config.Service, logger golog.Logger) (datamanager.Service, error) {
+	var syncLogger golog.Logger
```

Comment:
@edaniels checking if something like this is reasonable to limit logs.

Right now the majority of the spam is from the fix below (only log upload failure if `!errors.Is(uploadErr, context.Canceled)`), but any other errors (like a mismatch of super old data from a deprecated API) would also otherwise spam.

Reply:
yup fine!